browser-test/src/admin/admin_program_translation.test.ts

```diff
@@ -415,6 +440,20 @@ test.describe('Admin can manage program translations', () => {
         'Spanish block eligibility message',
       )
     })
-    // TODO: publish and verify the translation applicant-side
+
+    await test.step('Publish and verify the translation on applicant side', async () => {
+      await adminPrograms.publishProgram(programName)
+      await logout(page)
+
+      await loginAsTestUser(page)
+      await selectApplicantLanguage(page, 'Español')
+      await applicantQuestions.applyProgram('Spanish name')
+      await applicantQuestions.answerTextQuestion('ineligible')
+      await page.click('text="Guardar y continuar"')
+      await validateScreenshot(
```

Comment:
**suggestion:** Instead of adding a screenshot here, could we do the text validation similar to above?

Reply:
We could. I didn't find a single screenshot about ineligible block view so thought to add one to show what it looks like. Maybe it's stored somewhere and I just didn't find it.